KruBot/frmKruBot.cs

```diff
@@ -347,7 +375,75 @@ private void Client_OnMessageReceived(object sender, OnMessageReceivedArgs e)
                     GiveCurrency(command[1], GetCurrency(command[1]) * -1);
 
                 }
+                return;
             }
+
+            // Mod only commands
+            if (e.ChatMessage.UserType == TwitchLib.Client.Enums.UserType.Moderator)
```

Comment:
Also check if the user is the streamer - TwitchLib.Client.Enums.UserType.Streamer iirc

Reply:
Changed and tried it out, looks like it was UserType.Broadcaster.
Slipped my mind.